cotainr/tests/pack/test_conda_install.py

```diff
@@ -444,15 +448,55 @@ def test_installer_download_fail(
         self,
         patch_urllib_urlopen_force_fail,
         patch_disable_singularity_sandbox_subprocess_runner,
-        patch_disable_get_install_script,
     ):
-        with SingularitySandbox(base_image="my_base_image_6021") as sandbox:
+        with SingularitySandbox(
+            base_image="my_base_image_6021", architecture="x86_64"
+        ) as sandbox:
             with pytest.raises(
                 urllib.error.URLError, match="PATCH: urlopen error forced for url="
             ):
                 CondaInstall(sandbox=sandbox)
 
 
+class Test_GetInstallScript:
+    @pytest.mark.parametrize("arch", ["arm64", "aarch64"])
+    def test_arm_success(self, arch):
+        assert CondaInstall.get_install_script(arch) == "Miniforge3-Linux-aarch64.sh"
+
+    def test_x86_success(self):
+        assert CondaInstall.get_install_script("x86_64") == "Miniforge3-Linux-x86_64.sh"
+
+    def test_None_error(self):
+        with pytest.raises(
+            RuntimeError, match="which indicates that it has not been started correctly"
+        ):
+            CondaInstall.get_install_script(None)
+
+    @pytest.mark.parametrize(
+        "arch",
+        [
+            "test",
+            "arms64",
+            "aarchs64",
+            "None",
+            "Weird",
+            "WINDOWS",
+            "icecream",
+            "86_64",
+            "pc",
+        ],
+    )
+    def test_arm_success(self, arch):
```

Comment:
I guess this was supposed to be named differently, e.g.
```suggestion
    def test_unknown_arch_error(self, arch):
```